cob_twist_controller/include/cob_twist_controller/cob_twist_controller.h

```diff
@@ -59,6 +59,75 @@
 #include "cob_twist_controller/limiters/limiter.h"
 #include "cob_twist_controller/callback_data_mediator.h"
 
+#include <deque>
+
+class MovingAverage{
```

Comment:
please move '{' to next line to keep the style consistent with the rest of the package!